icechunk-python/python/icechunk/__init__.py

```diff
@@ -594,7 +566,12 @@ async def delete(self, key: str) -> None:
         ----------
         key : strz
         """
-        return await self._store.delete(key)
```

Comment:
I think it would be better to implement this behavior in zarr.rs

Reply:
Do you mean the `delete_dir` behavior?

Reply:
#390

Reply:
Sorry, no, I mean the fact that delete doesn't fail if the key doesn't exist. I think it's a behavior the rust store should implement